src/services/cube-handler.ts

```diff
@@ -575,7 +575,7 @@ export async function createDateDimension(quack: Database, extractor: object | n
   logger.debug(`Period coverage: ${zonedStartDate} to ${zonedEndDate}`);
 
   await quack.exec(`CREATE TABLE IF NOT EXISTS metadata (key VARCHAR, value VARCHAR);`);
-  const metaDataCoverage = await quack.all("SELECT * FROM metadata WHERE key = 'start_data' OR key = 'end_date';");
+  const metaDataCoverage = await quack.all("SELECT * FROM metadata WHERE key in ('start_data', 'end_date');");
```

Comment:
`start_data` should this be `start_date` ?

Reply:
Yes... Well caught... Now changed.